### PR TITLE
don't sync extraneous files(e.g. readme,owners)

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: roles/openshift_examples/examples-sync.sh [hunk 1]
```diff
@@ -9,7 +9,7 @@ XPAAS_VERSION=ose-v1.4.14
 RHDM70_VERSION=7.0.1.GA
 RHPAM70_VERSION=7.0.0.GA
 DG_72_VERSION=datagrid72-dev
-ORIGIN_VERSION=${1:-v3.9}
+ORIGIN_VERSION=${1:-v3.11}
```
Comment (bparees) [inline roles/openshift_examples/examples-sync.sh]: note i bumped this to 3.11 also.

**Comment by bparees** [comment]
@sdodson this also includes a "safe" sync of the upstream changes that we can pick up (i manually undid the changes we can't pick up from upstream)

**Comment by sdodson** [comment]
Thanks for taking care of all that.
/lgtm

**Comment by sdodson** [comment]
control plane flake on fedora, merging.

**Pull request merged**